src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+

```

Comment:
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#please-dont-add-redundant-empty-lines-to-your-code